.github/workflows/run_azure_dataspace_tests.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
 
       - name: 'Upgrade docker-compose (for --wait option)'
         run: |
-          sudo curl -L https://github.com/docker/compose/releases/download/v2.6.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+          sudo curl -L https://github.com/docker/compose/releases/download/v2.21.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
```

Comment:
now `compose` is part of the `docker` runtime and can be invoked with `docker compose`, so download and install docker-compose shouldn't be needed anymore, could you adapt this?